core-services/prow/02_config/_plugins.yaml

```diff
@@ -711,21 +711,25 @@ config_updater:
       clusters:
         app.ci:
         - ci
+      gzip: true
```

Comment:
Can we just set `gzip: true` in the global config_updater config and disable it in the cases where we need it disabled (if any)?

Reply:
I don't think that this is a good idea. All templates will break for sure and I don't know what else. I wonder is ci-operator can handle it. Let me check this first.

Reply:
Yep.. templates will break. https://github.com/openshift/ci-tools/blob/master/cmd/ci-operator/main.go#L506
I will check what else can break one by one.